test/e2e/karmadactl_test.go

```diff
@@ -522,3 +522,85 @@ var _ = ginkgo.Describe("Karmadactl exec testing", func() {
 		}
 	})
 })
+
+var _ = framework.SerialDescribe("Karmadactl top testing", ginkgo.Labels{NeedCreateCluster}, func() {
```

Comment:
I think we don't need `Serial`

Reply:
yeah~you are right! I will fix it right away